ddl/label/rule.go

```diff
@@ -88,7 +99,7 @@ func (r *Rule) Clone() *Rule {
 }
 
 // Reset will reset the label rule for a table/partition with a given ID and names.
-func (r *Rule) Reset(id int64, dbName, tableName string, partName ...string) *Rule {
+func (r *Rule) Reset(ids []int64, dbName, tableName string, partName ...string) *Rule {
```

Comment:
I think we can rewrite like `Reset(dbName, tableName string, partName string, ids ...int64)`, which seems more appropriate.

Reply:
partName is an option instead

Reply:
You can pass "", possibly `interface`.